tutorials/Cargo.toml

```diff
@@ -11,8 +11,10 @@ gstreamer = { path = "../gstreamer" }
 gstreamer-audio = { path = "../gstreamer-audio" }
 gstreamer-video = { path = "../gstreamer-video" }
 gstreamer-app = { path = "../gstreamer-app" }
+gstreamer-pbutils = { git = "https://github.com/sdroege/gstreamer-rs" }
```

Comment:
You would provide the path here, like in the lines above :)

Reply:
`../gstreamer-pbutils`

Reply:
Hah, ok. Is `cargo build` in the top-level dir supposed to work? I'm hitting what appears to be an unrelated error while testing the toml patch,

```
[cturner@localhost gstreamer-rs]$ pwd
/home/cturner/git/gstreamer-rs
[cturner@localhost gstreamer-rs]$ cargo build
    Updating crates.io index
    Updating git repository `https://github.com/gtk-rs/sys`
    Updating git repository `https://github.com/sdroege/gstreamer-sys`                                                         
    Updating git repository `https://github.com/gtk-rs/pangocairo`                                                             
error: no matching version `^0.5.0` found for package `glib-sys`                                                               
location searched: https://github.com/gtk-rs/sys
versions found: 0.7.0
required by package `glib v0.4.0 (https://github.com/gtk-rs/glib#a78f961b)`
    ... which is depended on by `gstreamer v0.13.0 (/home/cturner/git/gstreamer-rs/gstreamer)`
```

Reply:
`cargo build --all` and run a `cargo update` after you changed dependencies